### DISCUSSION TITLE
Welcome to fpdb-3 Discussions!

### DISCUSSION THREAD

**Discussion by jejellyroll-fr** [jejellyroll-fr/fpdb-3#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brunetton** [comment]
I think we should add a "development" category in discussions to separate technical communication from user point of view ones.

What do you think about @jejellyroll-fr  ?

**Comment by brunetton** [comment]
I'd like to contribute to the wiki. Can I have write access to the wiki Git ? 

![image](https://github.com/user-attachments/assets/6d87db12-c266-4db5-9404-2d6c9507958b)

We can keep branch organization if you want to review my changes of course, you could be the one who merge.

![](https://i.imgflip.com/94wgu5.jpg)

**Comment by jejellyroll-fr** [comment]
i give you access to the repo ;)
